tools/rosgraph/src/rosgraph/rosenv.py

```diff
@@ -69,7 +72,7 @@ def get_master_uri(env=None, argv=None):
                 if not val:
                     raise ValueError("__master remapping argument '%s' improperly specified"%arg)
                 return val
-        return env[ROS_MASTER_URI]
+        return env.get(ROS_MASTER_URI, DEFAULT_MASTER_URI)
     except KeyError as e:
```

Comment:
I assume there is no case left which might raise a `KeyError`. If that is the case please remove the `try` / `except`.